server/browserstack/browserstack.js

```diff
@@ -0,0 +1,50 @@
+const browserstackLocal = require('browserstack-local');
+const browserstack = require('browserstack');
+const open = require('open');
+const testBrowsers = require('./test-browsers.json');
+const url = process.argv[2];
+const user = process.env.BROWSERSTACK_USERNAME;
+const key = process.env.BROWSERSTACK_KEY;
```

Comment:
This might be nicer as a config file - I personally hate adding env variables as I use virtual envs where I can.